README.md

```diff
@@ -130,6 +130,9 @@ httpd = require('http.server').new(host, port[, { options } ])
 
   By default uses `log.info` function for requests logging.
 * `log_errors` - same as the `log_requests` option but is used for error messages logging. By default uses `log.error()` function.
+* `keepalive_disable` - disables keep-alive connections with misbehaving
```

Comment:
> What's the motivation behind this solution?

Simplicity. There was a problem on production with exactly a single HTTP client.
My pull request solves the problem described in a ticket.
Regarding choosing map or an array: the solution with map is much easy to implement. With array, as you correctly said you need to convert to a map. Then you need to put a map to cache and correctly invalidate cache because user can change `httpd.options.keepalive_disable` in runtime.

> Moreover, this solution should be thoughtfully verified with customer @vasiliy-t . I'm not sure it will satisfy Delivery team.

Sure, it should be. When we will merge this change to a master. Vasya needs a fix as a Luarocks release for fix validation.

Reply:
Well, when this will be merged to master and tagged, it would be too late to change implementation and verify anything

Reply:
Implementation of solution is LGTM. I can't say that I do not have doubts about solution design itself, so I propose to ask someone to review it additionally

Reply:
AFAIK, the result of the discussion with Delivery was to keep this feature for now and to make something more forward-thinking in future. I don't see any blocks for this PR. I leave the decision about closing #137 ticket with this PR and opening a new one for another feature (like killing idle keep-alive connections) or keeping #137 opened to author.